src/modules/user/user.controller.ts

```diff
@@ -103,9 +104,9 @@ export class UserController {
   @ApiCreatedResponse(responsedoc.addWishlist)
   async addWishlist(
     @Param('userId') userId: Schema.Types.ObjectId,
-    @Body() cId: Schema.Types.ObjectId,
+    @Body() createWishlistDto: CreateWishlistDTO,
```

Comment:
If you are just adding schecmaID then add it like that.

Reply:
The controller was well written before. It was service file which needs a revamp.